board_descriptors/0679be07-6106-48aa-8057-b1d4f2e18a99/board-nrf52840dk-SERIAL.yml

```diff
@@ -1,6 +1,10 @@
 model: nrf52840dk
-hw_rev: '3.2'
-serial_number: 0xfoobar
+serial_number: '0679be07-6106-48aa-8057-b1d4f2e18a99'
```

Comment:
I don't think `serial_number` is a good label here. This is just the host ID, no?